templates/style.scss

```diff
@@ -98,6 +98,7 @@ body {
 }
 
 pre {
+    font-size: 0.8rem;
```

Comment:
I had a chance to test this today. `0.8rem` seems a little small, personally; what do you think about `0.9rem`? It still fits 100 characters, but is a little bigger and easier to read.